src/resultscallercalleepage.cpp

```diff
@@ -38,17 +38,30 @@
 #include "resultsutil.h"
 
 #include "models/callercalleemodel.h"
+#include "models/callercalleproxy.h"
 #include "models/costdelegate.h"
 #include "models/filterandzoomstack.h"
 #include "models/hashmodel.h"
+#include "models/sourcemapproxy.h"
 #include "models/treemodel.h"
 
 namespace {
+QSortFilterProxyModel* getProxy(SourceMapModel* model)
```

Comment:
no `get` prefix please - also this is not a getter, but a factory. rename to `createProxy`?